### ISSUE TITLE
Want to show Current,Previous and Next Months Data

### ISSUE THREAD

**Issue by Bikram786** [patchthecode/JTAppleCalendar#843]
Hello Sir,

Hope you are doing well!

First of all thanks a lot for such a beautiful library for the calendar. I am facing problem in this i.e. 

I am using your demo project and I saw that you used starting and end dates for set the calendar and in demo project everytime show the first month data that we set starting date but I want to see the current month and I want to scroll previous and next month but when a user open or see calendar every time he can see the current month and dates so I am unable to do this.

Please help me I am waiting for your positive response.

Thanks & Best Regards,
Bikramjeet Singh

**Comment by patchthecode** [comment]
Im sorry, but i cannot understand what you are asking.
Can you rephrase the question?

**Comment by Bikram786** [comment]
Thanks a lot for your response.

Sir, I am asking that if you saw the FSCalander or used FSCalander. In this, we no need to set the start and end date of the calendar and it automatically shows the current, previous and next months calendar. So how I can achieve this type of functionality by using JTAppleCalendar.

Hope you understand.

Thanks

**Comment by patchthecode** [comment]
I understand.

1. In this calendar library, he need a start and end date. This sets the limit to where the calendar can scroll to. If you need infinite scrolling then you will have to implement this.

2. If you need **previous** and **next** functionality, this is easy.
create two buttons and then call, 
```swift
calendarView.scrollToSegment(.next)
calendarView.scrollToSegment(.previous)
calendarView.scrollToSegment(.start)
calendarView.scrollToSegment(.end)
```

**Comment by Bikram786** [comment]
Thanks for your response.

But below I explained actually what functionality I want:- 

Step 1:- Suppose I used JTAppleCalendar.

Step2:- I gave the start and end date of the calendar.

Step3:- Now When I open the calendar. Every time calendar starts from the started date which I have given.

Step4:- But I want that calendar start from the current date and I can scroll previous and next dates.

Step5:- The main problem I faced how I set or shows the current date on the calendar.

Hope you understand.

Thanks & Best Regards,
Bikramjeet Singh

**Comment by patchthecode** [comment]
if you want the calendar to start at the current date, then do the this inside your `viewDidLoad()` function

```swift
calendarView.scrollToDate( Date() )
```

**Comment by Bikram786** [comment]
Yes, I can do this. Most of my problem is solved.

Now only one thing is left i.e.:- 

I am sending you below image:- 

I want to show cell height according to its content and I want to show full the calendar. Now If I set a static height of calendar and increased the size of a cell. I unable to see all content so please help me.

![simulator screen shot - iphone 6s - 2018-08-07 at 10 27 58](https://user-images.githubusercontent.com/29310008/43755241-a709cf88-9a2c-11e8-882a-cef08baac2f8.png)

Thanks

**Comment by patchthecode** [comment]
With the library, the height of the cell is not determined by the height of the text in the cell.
The height of the cell is determined by the `Height of the calendar`.

If you need you cell to be a particular height, then calculate what is the maximum height of one cell is, then multiply it by 6 (because you have a 6 row calendar) --> 
```
CalendarViewHeight = CellHeight * 6
```

**Comment by patchthecode** [comment]
closing issue.
If you still need help with this. then let me know.
